src/main.ts

```diff
@@ -32,6 +33,19 @@ export enum ReturnCode {
   InternalError
 }
 
+export async function initSasjs() {
+  return await init()
+    .then(() => {
+      displaySuccess(
+        'Project is powered with SASjs. You can use any sasjs command, now.'
```

Comment:
I'd suggest something along the lines of - `This project is powered by SASjs. You can now use any sasjs command within the project. For more information, type 'sasjs help' or visit https://cli.sasjs.io/`

@allanbowe what do you think?